src/settings_internal.h

```diff
@@ -87,8 +87,7 @@ enum SettingType {
 typedef bool OnChange(int32 var);           ///< callback prototype on data modification
 typedef size_t OnConvert(const char *value); ///< callback prototype for conversion error
 
-/** Properties of config file settings. */
-struct SettingDescBase {
+struct SettingDesc {
```

Comment:
The doxygen felt off.

And basically this change is a result of the commit before this one, right? Nobody is using `SettingDescBase` anymore, so we can drop it?